.github/workflows/production.yml

```diff
@@ -0,0 +1,72 @@
+name: Production
+
+# trigger this deployment by running:
+# gh api repos/bcgov/wps/deployments -f ref="[your branch]"
+on:
+  # reference used to figure this part out:
+  # https://docs.github.com/en/actions/reference/events-that-trigger-workflows#deployment
+  deployment:
+
+jobs:
+  deploy_to_production:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Find PR number
+        uses: actions/github-script@v4
+        with:
+          # reference used to figure out this part:
+          # https://docs.github.com/en/rest/reference/pulls#list-pull-requests
+          # https://octokit.github.io/rest.js/v18#pulls
+          # https://github.com/actions/github-script
+          script: |
+            // get pull requests relating to this commit hash
+            const pulls = (await github.repos.listPullRequestsAssociatedWithCommit({owner: context.repo.owner, repo: context.repo.repo, commit_sha: context.sha }))
+            console.log(pulls)
+            // find an open PR (potentially problematic if you have multiple PR's open!)
```

Comment:
You can get this exact PR number from an environment variable, check out: https://github.com/bcgov/wps/blob/lnd/pr-github-action/.github/actions/pr-test-url/index.js

Reply:
It's not common to have multiple PR's at the same commit point. Thanks for the link - I'll test.

Reply:
@conbrad so actually that doesn't work - loading that file is the same as referencing context - and it doesn't have the PR in this context.

Reply:
Oops, sorry about that.